### PR TITLE
feat: Hangman Game java based project

### PR DESCRIPTION
**Pull request by mradulrathore**

### CODE REVIEW

File: projects/mradulrathore/HangmanGame.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+<Overview>
+
+**Hangman Game**
+---               
+It is a simple Java-based hangman game.The player has to guess the hidden word by guessing the individual characters of the word.
+
+To see how it looks like view images in image folder.
+
+To run it, download the files and run the Hangman.java file
```
Comment (ramonh) [inline projects/mradulrathore/HangmanGame.mdx:28]: Could you perhaps please provide an example command for running this?

I'm not entirely familiar with running Java files, and as a learner would appreciate reproducible steps for running the game.

**Review by ramonh — CHANGES_REQUESTED** [review]
Thank you so much for your contribution, @mradulrathore!

I left one request in the pull request, but otherwise looking good!

File: projects/mradulrathore/HangmanGame.mdx (hunk 1 above)
Comment (mradulrathore) [reply projects/mradulrathore/HangmanGame.mdx]: thanks for reviewing! I've added the description for running the application.

**Review by ramonh — APPROVED** [review]
Looks great, thank you!

I am terrible at this, and have lost three times in a row 😂 

Welcome to the port!

**Comment by mradulrathore** [comment]
I can relate. Sometimes it happens with me as well😂
Thank you for the warm welcome and looking forward to contributing!

**Pull request merged**